connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -3233,14 +3233,15 @@ class PlanGenerationTestSuite
     "connect/common/src/test/resources/protobuf-tests/common.desc"
 
   test("from_protobuf messageClassName") {
```

Comment:
If we deem this plan too intricate, we could delete or ignore test cases `from_protobuf messageClassName` and `from_protobuf messageClassName options` to temporarily abandon this testing scenario

Reply:
This is great! Thank you.